### DISCUSSION TITLE
Grafana future on SO2.4?

### DISCUSSION THREAD

**Discussion by TotieBash** [Security-Onion-Solutions/securityonion#10098] [2.4]
I notice Grafana is not in the 2.4 Documentation and I see screenshot of native Influxdb2 webui.  I  am wondering if the Grafana docker container will make the cut for 2.4 or will it be remove.  Thanks.

**Comment by cm-ops** [comment]
Grafana will not be in 2.4, it will use Influxdb 2 instead.

**Reply by TotieBash** [reply]
=[  ...  Ok thanks for the response...
